tests/pds/dataservice_test.go

```diff
@@ -567,7 +567,9 @@ var _ = Describe("{ScaleDownScaleupPXCluster}", func() {
 					nodes, err := pdslib.GetNodesOfSS(*deployment.ClusterResourceName, namespace)
 					log.FailOnError(err, "Error while getting Data Serice Nodes")
 					deploymentNode := nodes[0]
-					nodeList = append(nodeList, deploymentNode)
+					if len(nodeList) < 2 {
+						nodeList = append(nodeList, deploymentNode)
+					}
```

Comment:
How will you handle the case when the `nodeList` is greater than 2?

Reply:
We are creating the nodeList here. So we are selecting how many nodes to shutdown Px on. For eg, We have 5 worker nodes, Then if we shutdown 3 px nodes, we do not have enough nodes with volume replicas and the pods go into pending state.
with this we are only controlling how many nodes we shut down Px on